tests/apps/courses/test_models_course_run.py

```diff
@@ -609,15 +610,16 @@ def test_models_course_run_mark_dirty_any_field(self):
                 CourseRun._meta.fields,
             ),
         )
+        stub = CourseRunFactory(
+            sync_mode="manual",
+            catalog_visibility=CourseRunCatalogVisibility.COURSE_ONLY,
```

Comment:
why fix this field? It should not be necessary?

Reply:
@sampaccoud  I don't know why, but when I remove that line, the test fails with:
```python
            self.assertEqual(
                title_obj.publisher_state,
                PUBLISHER_STATE_DIRTY,
>               msg=f"After refreshing from db {field:s}",
            )
E           AssertionError: 0 != 1 : After refreshing from db catalog_visibility
```

Can you help me with this?

Reply:
ok I checked and sorry, it has to be modified from the default value so what you do is correct :+1: